changelog.md

```diff
@@ -1,9 +1,18 @@
+## 3.4.0 / 2020-12-21
 
+### New Features
+
+- Files uploaded to Tracim can now be stored on a Amazon S3 compatible server, see [backend documentetation](backend/doc/setting.md) for details on how to configure it.
```

Comment:
```suggestion
- Files uploaded to Tracim can now be stored on a Amazon S3 compatible server, see [backend documentation](backend/doc/setting.md) for details on how to configure it.
```
maybe but some warning about the fact the feature is in some kind of "beta" stage ? (as preview are not efficiently generated with it for now ?)

Reply:
Added a sentence to explain this.